api/src/main/java/io/grpc/Server.java

```diff
@@ -102,6 +104,8 @@ public int getPort() {
 
   /**
    * Initiates an orderly shutdown in which preexisting calls continue but new calls are rejected.
+   * After calling this method, clients may no longer connect to the listening socket(s).
```

Comment:
Nit: I feel like this first sentence is mostly covered by the first. And the user couldn't really notice if we accepted another connection anyway. How about "After this call returns, this server has released the listening socket(s) and it may be reused by another server."

Reply:
Done.